Assets/Scripts/Office/VendingMachine/VendingMachineTrigger.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class VendingMachineTrigger : MonoBehaviour
+{
+    private CanvasManager canvasManager;
+
+    private void Start()
+    {
+        canvasManager = GetComponentInParent<CanvasManager>();
+    }
+
+    private void OnTriggerStay(Collider other)
```

Comment:
this seems so inefficient, and i feel like we're gonna use this a lot more. I can't think of a better way to do it, but we will probs wanna find a way